pkg/render/amazoncloudintegration_test.go

```diff
@@ -74,7 +81,14 @@ var _ = Describe("AmazonCloudIntegration rendering tests", func() {
 
 	It("should render an AmazonCloudConfiguration with specified configuration", func() {
 		// AmazonCloudIntegration(aci *operatorv1.AmazonCloudIntegration, installation *operator.Installation, cred *AmazonCredential, ps []*corev1.Secret, openshift bool) (Component, error) {
-		component, err := render.AmazonCloudIntegration(instance, installation, credential, nil, openshift)
+		cfg := &render.AmazonCloudIntegrationConfiguration{
```

Comment:
Similar to my comment for apiserver, I think you should have a cfg be created in the BeforeEach so the tests can use defaults and only need to set any specific fields they need to tweak.

Reply:
Moved cfg to BeforeEach section